python/sanic/server_python_sanic.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3.5
+# -*- coding: utf-8 -*-
+
+from sanic import Sanic 
```

Comment:
When I try this, I got following error.
May be you should put requirement.txt at the directory and install sanic in Makefile?
```
ImportError: No module named 'sanic'
```